text/0124-documents-api.md

```diff
@@ -51,11 +55,12 @@ Unique identifier of an index.
 
 ##### 3.1.1.2. Query Parameters
 
-| Field                    | Type                     | Required |
-|--------------------------|--------------------------|----------|
-| `offset`                 | Integer / `null`         | false    |
-| `limit`                  | String / `null`          | false    |
-| `fields`                 | String / `null`          | false    |
+| Field                    | Type                      | Required |
+|--------------------------|---------------------------|----------|
+| `offset`                 | Integer / `null`          | false    |
+| `limit`                  | String / `null`           | false    |
```

Comment:
```suggestion
| `offset`                 | Integer / `null`          | false    |
| `limit`                  | Integer / `null`           | false    |
```
Using `null` or a string for `offset` and `limit` returns: 
```
Invalid value type at `.offset`: expected a positive integer
```
Should users be allowed to use null here?

Reply:
Oh yes, you're right; my bad, sorry!
Only the filter and fields can be null.